introspect.py

```diff
@@ -237,16 +237,33 @@ def matched_adjacencies(self, node_type, fq_name_str):
         neighbors = url_dict_resp['ifmap_db'][0]['neighbors']
         return neighbors
 
-    def get_routes(self, vrf_fq_name):
-        url_path = 'Snh_ShowRouteReq?x=%s.inet.0'%vrf_fq_name
+    def get_routes(self, vrf_fq_name, url_path):
         return self.get(path=url_path)
 
-    def is_route_exists(self, vrf_fq_name, address):
-        routes = self.get_routes(vrf_fq_name)
+    def is_route_exists(self, vrf_fq_name, address, rt_type):
+        if rt_type == 'inet':
+            rt_name = 'inet'
+        elif rt_type == 'inet6':
+            rt_name = 'inet6'
+        elif ((rt_type == 'bridge') or (rt_type == 'evpn')):
+            rt_name = 'evpn'
+        else:
+            rt_name = None
```

Comment:
have added the following changes to handle this case
        else:
            rt_name = None
            routes = None
            return(False, routes)

With out this change also, existing code returns the False and behavior is same. How to do increment commit to accommodate this change? Does it require one more pull request?